src/core/core.scale.js

```diff
@@ -331,6 +335,30 @@ var Scale = Element.extend({
 		return me.minSize;
 
 	},
+
+	/**
+	 * @private
+	 */
+	_configure: function() {
+		var me = this;
+		var reversePixels = me.options.ticks.reverse;
+		var startPixel, endPixel;
+
+		if (me.isHorizontal()) {
+			startPixel = me.left;
+			endPixel = me.right;
+		} else {
+			startPixel = me.top;
+			endPixel = me.bottom;
+			// by default vertical scales are from bottom to top, so pixels are reversed
+			reversePixels = !reversePixels;
+		}
+		me._startPixel = startPixel;
+		me._endPixel = endPixel;
+		me._reversePixels = reversePixels;
+		me._lengthPixels = endPixel - startPixel;
```

Comment:
Thanks for clarifying the names. I'd be ok just calling this `_length` still. I had assumed it was the size in pixels since most sizes in the library are pixels. I just got a bit confused about start meaning canvas origin

Reply:
done